pkg/kubelet/dockershim/docker_streaming.go

```diff
@@ -78,8 +80,8 @@ func (ds *dockerService) ExecSync(_ context.Context, req *runtimeapi.ExecSyncReq
 	var stdoutBuffer, stderrBuffer bytes.Buffer
 	err := ds.streamingRuntime.exec(req.ContainerId, req.Cmd,
 		nil, // in
-		ioutils.WriteCloserWrapper(&stdoutBuffer),
-		ioutils.WriteCloserWrapper(&stderrBuffer),
+		ioutils.WriteCloserWrapper(ioutils.LimitWriter(&stdoutBuffer, maxMsgSize)),
```

Comment:
@tallclair we cannot pass through a size parameter to `ExecSync` from `RunInContainer` as that would mean a change in signature of `ExecSync`. So i just set the LimitWriter to the same 16MB (that is the limit we set for GPRC for remote CRIs)

Reply:
That makes sense to me.